### DISCUSSION TITLE
Recurring Scheduled Tickets

### DISCUSSION THREAD

**Discussion by Wxam** [polonel/trudesk#548] [Ideas]
It would be really nice to have an option to have it so that we can make recurring scheduled tickets.

This would be usefull for maintenance tickets for like updates or something like that.

**Comment by polonel** [comment]
I think this would be a great feature. I'm in the process of converting some of the outdated core over to modern JS & React. Once I complete this I'll schedule this feature.

**Comment by Wxam** [comment]
It maybe could look a little like in the picture
![image](https://user-images.githubusercontent.com/20145041/184524971-47b9a602-8afe-4610-97fa-06b5ba41f35f.png)

Note: This is not mine! I got this from Halo ITSM
